# Goal
float32 x
float32 z
float32 roll
float32 pitch
float32 moving_time
float32 wp_moving_time
float32 wp_accel_time
float32 wp_period
---
# Result
string log
bool success
---
# Feedback
string log
